subsys/bluetooth/audio/endpoint.h

```diff
@@ -77,7 +77,7 @@ struct bt_audio_broadcast_source {
 	struct bt_iso_chan *bis[BROADCAST_STREAM_CNT];
 	struct bt_codec_qos *qos;
 	/* The streams used to create the broadcast source */
-	struct bt_audio_stream **streams;
+	sys_slist_t streams;
```

Comment:
Where's `sys_slist_init`?

Reply:
`sys_slist_init` is defined as 
```c
static inline void sys_slist_init(sys_slist_t *list)
{
	list->head = NULL;
	list->tail = NULL;
}
```

and since the `bt_audio_broadcast_source`s and `bt_audio_broadcast_sink`s are statically defined, as well as `memset` on deletion, `sys_slist_init` provides no value. 

I can, however, still add it as part of the `create` function for semantic reasons if you like.